src/paginator.ts

```diff
@@ -31,7 +33,10 @@ export class Paginator<Params, Result>
       params: params ?? this.nextParams,
     });
 
-    this.nextUrl = this.pluckNext(response.headers?.link as string);
+    this.nextUrl =
+      typeof response.headers?.Link === 'string'
```

Comment:
Capital `Link` doesn't work with Axios because they normalize it to lowercase.
https://github.com/axios/axios#response-schema

Probably we need an additional layer that makes Header case insensitive